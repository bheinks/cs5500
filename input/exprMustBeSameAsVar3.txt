(* 
  Expression must be of same type as variable
*)

program p;

var i : integer;

begin
  i := i >= 99;
  write('!')
end.

