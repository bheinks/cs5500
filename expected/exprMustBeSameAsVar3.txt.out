Line 10: Expression must be of same type as variable
